build-logic/module-identity/src/main/kotlin/gradlebuild/identity/extension/ModuleIdentityExtension.kt

```diff
@@ -37,23 +35,6 @@ abstract class ModuleIdentityExtension(val tasks: TaskContainer, val objects: Ob
     abstract val snapshot: Property<Boolean>
```

Comment:
Is there a `ModuleIdentityExtension` for each project? Does this make sense or should there be only one `ModuleIdentityExtension`?

Reply:
I think so, and it seems unnecessary... But I'd not change this in this PR.